opensciencegrid/vo-frontend/.github/workflows/build-sw-container.yml

```diff
@@ -0,0 +1,73 @@
+name: Build and Push Docker image
```

Comment:
You can drop this file completely, we use what's in `.github` of the root of this dir now

Reply:
Done.